packages/@aws-cdk/aws-autoscaling/README.md

```diff
@@ -23,32 +23,47 @@ new autoscaling.AutoScalingGroup(stack, 'ASG', {
 > internet) which is set to `true` by default. Be sure to set this to `false`  if you don't want
 > your instances to be able to start arbitrary connections.
 
-### AMIs
+### Machine Images (AMIs)
 
-AMIs control the OS that gets launched when you start your instance.
+AMIs control the OS that gets launched when you start your instances. The constructs
+for selecting AMIS are in the `@aws-cdk/aws-ec2` package.
 
 Depending on the type of AMI, you select it a different way.
 
-The latest version of Windows images are regionally published under labels,
-so you can select Windows images like this:
+The latest version of Amazon Linux and Microsoft Windows images are
+selectable by instantiating one of these classes:
 
-    new ec2.WindowsImage(WindowsVersion.WindowsServer2016EnglishNanoBase)
-
-You can select the latest Amazon Linux image like this:
-
-    new ec2.AmazonLinuxImage()
+```ts
+// Pick a Windows edition to use
+const windows = new ec2.WindowsImage(WindowsVersion.WindowsServer2016EnglishNanoBase);
```

Comment:
Wouldn't it be sweet if this was a `.lit.ts` instead? Same applies to further code excerpts. I'm becoming wary of not realizing when we break those examples... literate examples allow us to make sure they at the very least build.